tgui/packages/tgui/styles/interfaces/CrewManifest.scss

```diff
@@ -0,0 +1,57 @@
+@use '../colors.scss';
+
+$department_map: (
```

Comment:
this already exists elsewhere and is used by the prefs menu, they should share it if possible.